ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/datepicker.html

```diff
@@ -34,22 +34,24 @@
      <div class="cmp-adaptiveform-datepicker__label-container">
         <div data-sly-call="${label.label @componentId=widgetId, labelValue=datePicker.label.value, labelVisible=datePicker.label.visible, labelRichText=datePicker.label.richText, bemBlock='cmp-adaptiveform-datepicker'}" data-sly-unwrap></div>
         <div data-sly-call="${questionMark.questionMark @componentId=datePicker.id, longDescription=datePicker.description, bemBlock='cmp-adaptiveform-datepicker'}" data-sly-unwrap></div>
-     </div>
-     <input type="date"
-            name="${datePicker.name}"
-            class="cmp-adaptiveform-datepicker__widget"
-            disabled="${!datePicker.enabled}"
-            readonly="${datePicker.readOnly}"
-            required="${datePicker.required}"
-            value="${datePicker.default? datePicker.default : ''}"
-            data-cmp-data-layer="${datePicker.data.json}"
-            title="${datePicker.tooltipVisible ? '' : datePicker.tooltipText}"
-            id="${widgetId}"
-            placeholder="${datePicker.placeHolder}"
-            min="${datePicker.minimumDate}"
-            max="${datePicker.maximumDate}"/>
-     <div data-sly-call="${shortDescription.shortDescription @componentId=datePicker.id, shortDescriptionVisible=datePicker.tooltipVisible, shortDescription=datePicker.tooltip, bemBlock='cmp-adaptiveform-datepicker'}" data-sly-unwrap>
-     </div>
-     <div data-sly-call="${longDescription.longDescription @componentId=datePicker.id, longDescription=datePicker.description, bemBlock='cmp-adaptiveform-datepicker'}" data-sly-unwrap></div>
-     <div data-sly-call="${errorMessage.errorMessage @componentId=datePicker.id, bemBlock='cmp-adaptiveform-datepicker'}" data-sly-unwrap></div>
-</div>
\ No newline at end of file
+    </div>
+    <sly data-sly-use.minDate="${'dateformatter.js' @date=datePicker.minimumDate}"
+         data-sly-use.maxDate="${'dateformatter.js' @date=datePicker.maximumDate}"></sly>
+    <input type="date"
+           name="${datePicker.name}"
+           class="cmp-adaptiveform-datepicker__widget"
+           disabled="${!datePicker.enabled}"
+           readonly="${datePicker.readOnly}"
+           required="${datePicker.required}"
+           value="${datePicker.default? datePicker.default : ''}"
+           data-cmp-data-layer="${datePicker.data.json}"
+           title="${datePicker.tooltipVisible ? '' : datePicker.tooltipText}"
+           id="${widgetId}"
+           placeholder="${datePicker.placeHolder}"
+           min="${minDate.formatDate}"
```

Comment:
Why cant you do this via java code itself ? We don't follow this pattern of JS execution in core component

Reply:
Just to be clear, since this is a sightly, formatting can't be done inside it, you are suggesting we create a new version for date picker interface and its implementation for this bug fix?